API/api/tests/test_event.py

```
@@ -1,6 +1,117 @@
+            event=self.event,
+            name="Matin",
+            start_time=time(9, 0),
+            end_time=time(12, 0),
+            slot_duration_minutes=30,
+        )
+
+        try:
+            self.update_event_url = reverse(
+                "update_event_date", kwargs={"event_id": self.event.id}
+            )
+        except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)